### PR TITLE
chore: update codeowners, readme

### PR DESCRIPTION
**Pull request by raphael-arce**

### CODE REVIEW

**Comment by raphael-arce** [comment]
@all-contributors please add @aeschi for code, design, review, ideas

**Comment by raphael-arce** [comment]
@all-contributors please add @raphael-arce for review, ideas

**Comment by raphael-arce** [comment]
@all-contributors please add @raphael-arce for review, ideas

File: README.md [hunk 1]
```diff
@@ -1,88 +1,78 @@
...
 
 ## Prerequisites
 
-Please have a look at the main repository: [https://github.com/technologiestiftung/idea-machine](https://github.com/technologiestiftung/idea-machine)
+These are the other necessary repositories: 
+- [https://github.com/technologiestiftung/idea-machine-frontend](https://github.com/technologiestiftung/idea-machine-frontend)
+- [https://github.com/technologiestiftung/idea-machine-dices](https://github.com/technologiestiftung/idea-machine-dices)
 
-## Installation
+If you want to run this yourself, you also need a supabase project (to store ideas)and
```
Comment (aeschi) [inline README.md]: space missing after ideas)

**Review by aeschi — APPROVED** [review]
One small typo otherwise LGTM! 👍

File: README.md (hunk 1 above)
Comment (raphael-arce) [reply README.md]: good catch, thank you 👍

**Pull request merged**